src/js/components/tasks/table.tsx

```diff
@@ -50,9 +50,9 @@ interface TableCellProps {
 interface Props {
   projectId: string;
   projectSlug: string;
-  epicSlug: string;
+  epicSlug?: string;
   tasks: Task[];
-  epicUsers: GitHubUser[];
+  epicUsers?: GitHubUser[];
```

Comment:
We'll want to add a new "Epic" column to this table, only for this use on the project-detail page.